scripts/publish-docker-images

```diff
@@ -21,8 +21,8 @@ IMAGE_NAME="amazon/amazon-ecs-agent"
 
 AWS_PROFILE=""
 STAGE_S3_BUCKET=""
-
-source $(dirname "${0}")/publishing-common.sh
+S3_ACL_OVERRIDE=""
```

Comment:
What is this used for?

Reply:
This variable is used in `publishing-common.sh->s3_cp`. We need `acl` to be `public_read` in that function.